### DISCUSSION TITLE
How do I use enumerations.

### DISCUSSION THREAD

**Discussion by McPhir** [iewnfod/CAIE_Code#87] [Q&A] [answered]
For example (like the example in the guide).
I am trying t odo this:
Type Season = (Spring,Summer,Winter,Autumn)
DECLARE ThisSeason : Season
ThisSeason <- Season.Spring
OUTPUT ThisSeason

//not a complaint. The extension is very good.

**Comment by iewnfod** [comment]
This might be a bug in for enumerate type. We will try to fix it.

**Comment by lightumcc** [comment, marked as answer]
Hi, this issue has been fixed in https://github.com/iewnfod/CAIE_Code/commit/67fd2c7ccd2ab407344730beae2ec66da5a167f3 on branch `dev`.
